README.md

```diff
@@ -0,0 +1,3 @@
+# Openshift Dedicated Adoption Metrics Exporter
+
+A prometheus exporter to expose metrics about various features used in Openshift Dedicated Clusters
```

Comment:
I'll update the README. I'm using using the controller-runtime because it simplifies some things. And this probably should not be called an operator. But we will using the OLM to distribute it. :man_shrugging: